migrate/20250128_make_lbvms_id_not_null.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    alter_table(:load_balancers_vms) do
+      set_column_not_null :id
```

Comment:
how do you backfill the id of existing load_balancers_vms?

Reply:
After the first commit is deployed, I'll go to production and update `id`s

Reply:
that's alright with me. a better approach might be to have another migration that executes those steps before setting the column to not null.